app/views/proposals/index.html.erb

```diff
@@ -15,4 +15,5 @@
     <dt>Valor total da locação: </dt>
     <dd> <%= proposal.total_amount %> </dd>
   </dl>
-<% end%>
\ No newline at end of file
+<% end%>
+<%= render 'proposals/proposals_user' %>
```

Comment:
Se a visualização de Corretor e Usuário não for diferente você pode usar a partial de collection:

http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections